backend/dataall/db/api/environment.py

```diff
@@ -1011,3 +1003,8 @@ def check_group_environment_permission(
     @staticmethod
     def get_environment_parameters(session, env_uri):
         return EnvironmentParameterRepository(session).get_params(env_uri)
+
+    @staticmethod
+    def get_boolean_env_param(session, env: models.Environment, param: str) -> bool:
```

Comment:
I saw you applied the usage of this in notebooks and datasets.
There is also one in [datapipelines](https://github.com/awslabs/aws-dataall/blob/modularization-main/backend/dataall/modules/datapipelines/services/datapipelines_service.py#L44) and [mlstudio](https://github.com/awslabs/aws-dataall/blob/modularization-main/backend/dataall/modules/mlstudio/services/mlstudio_service.py#L75) as well if you want to update those too.

Reply:
I had created this before the mlstudio and datapipelines were merged. I will update it!